tests/node/run.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+set -xe
+# We need to use absolute path for the Docker container
+# So make sure we're in the right WD
+cd -- `dirname ${BASH_SOURCE[0]}`
+
+TEST_DIRS=$(find . -mindepth 1 -maxdepth 1 -type d -print)
+ROOT_DIR="../.."
+
+export SEBAK_GENESIS=GDIRF4UWPACXPPI4GW7CMTACTCNDIKJEHZK44RITZB4TD3YUM6CCVNGJ
+export SEBAK_COMMON=GDYIHSHMDXJ4MXE35N4IMNC2X3Q3F665C5EX2JWHHCUW2PCFVXIFEE2C
+
+# We can only have one trap active at a time, so just save the IDs of containers we started.
+# Single quotes around  trap ensure that the variable is evaluated at exit time.
+DOCKER_CONTAINERS=""
+function dumpLogsAndCleanup () {
+    if [ ! -z "${DOCKER_CONTAINERS}" ]; then
+        for CONTAINER in ${DOCKER_CONTAINERS}; do
+            docker logs ${CONTAINER} || true
+        done
+        docker rm -f ${DOCKER_CONTAINERS} || true
+    fi
+}
+
+trap dumpLogsAndCleanup EXIT
+
+## Build the node runner docker image
+IMAGE=$(docker build --tag sebak:runner -q \
+    --build-arg BUILD_MODE="test" \
+    --build-arg BUILD_PKG="./cmd/sebak" \
+    --build-arg BUILD_ARGS="-coverpkg=./... -tags integration -c -o /go/bin/sebak"  \
+    ${ROOT_DIR}/ | cut -d: -f2)
+
+if [ -z ${IMAGE} ]; then
+    echo "Failed to build node runner docker image" >&2
+    exit 1
+fi
+
+
+# Build the integration test docker image
+IMAGE=$(docker build --tag sebak:api_tester -q \
```

Comment:
This file has a lot of modifications that shouldn't be there (it should be simply moved). Can you revert those ?

Reply:
sure.